provider/aws/lambda/cfevents/main.go

```diff
@@ -155,3 +133,18 @@ func parseMessage(msg string) (Message, error) {
 
 	return m, nil
 }
+
+func getLogGroup(stack string) (string, error) {
+	res, err := cf.DescribeStackResources(&cloudformation.DescribeStackResourcesInput{
```

Comment:
moving this off of DescribeStacks does seem like an improvement. But do you expect DescribeStackResources to throttle instead?

There are strategies of caching this stuff on the lambda file system if this isn't good enough.